precompiles/utils/macro/src/precompile/expand.rs

```diff
@@ -522,3 +522,4 @@ impl Precompile {
 		}
 	}
 }
+// TODO: Take precompile-utils from frontier
```

Comment:
Still relevant?

Reply:
Yes, the message is relevant indeed but I just put it to try something. So, removing... :)